ReverseProxy.md

```diff
@@ -0,0 +1,129 @@
+# WoT reverse proxy
+
```

Comment:
It would be better to call it a WoT directory (drawing parallel to [IETF resource directory](https://datatracker.ietf.org/doc/draft-ietf-core-resource-directory/)), or at least also mention  by this name.